Tools/ftscalingbench/ftscalingbench.py

```diff
@@ -54,8 +54,16 @@ def object_cfunction():
 
 @register_benchmark
 def cmodule_function():
+    N = 1000 * WORK_SCALE
+    for i in range(N):
+        math.cos(i / N)
+
+@register_benchmark
+def object_lookup_special():
+    # round() uses `_PyObject_LookupSpecial()` internally.
+    N = 1000 * WORK_SCALE
     for i in range(1000 * WORK_SCALE):
```

Comment:
Not sure if this is intentionally different from what's in `cmodule_function`, but if not:
```suggestion
    for i in range(N):
```